monoio/src/task/state.rs

```diff
@@ -80,6 +80,7 @@ impl State {
         Snapshot(self.0.load(Acquire))
     }
 
+    #[allow(dead_code)]
```

Comment:
Better to remove if it's dead.

Reply:
Agreed. If there's no any usage, it should be removed. However, if it is used under some condition(some `#cfg`), this is acceptable for me.

Reply:
The dead code is now removed.